src/Hosting/test/FunctionalTests/WebHostBuilderTests.cs

```diff
@@ -28,7 +29,8 @@ public async Task InjectedStartup_DefaultApplicationNameIsEntryAssembly(TestVari
             {
                 var logger = loggerFactory.CreateLogger(nameof(InjectedStartup_DefaultApplicationNameIsEntryAssembly));
 
-                var applicationPath = Path.Combine(TestPathUtilities.GetSolutionRootDirectory("Hosting"), "test", "testassets", "IStartupInjectionAssemblyName");
+                var baseDirectory = SkipOnHelixAttribute.OnHelix() ? AppContext.BaseDirectory : Path.Combine(TestPathUtilities.GetSolutionRootDirectory("Hosting"), "test", "testassets");
```

Comment:
Calling the skip attribute as a helper library? Wow.

Reply:
Yeah need a centralized helper for sure, the universe of these tests that need to do different stuff on helix will drive that, its in a shared source file for now, so it was easiest to just expose it there as a starting point.  Note: I'm just opening these PRs as starting points for the feature teams to be investigate.  These are 'repro' PRs